catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -285,4 +313,64 @@ public boolean matches(Object o) {
             return test.test((QueryRequest) o);
         }
     }
+
+    private void mockMultipleCatalogFrameworkQueries(int pageSize, int resourceLimit)
+            throws Exception {
+
+        mockResourceResponse();
+        QueryResponse queryResponseMock = mock(QueryResponse.class);
+
+        // Populate list of mock results, sized at resource limit
+        List<Result> resultsList = populateResultMockList(resourceLimit);
+        List<Result>[] queriedResultsList = mockQueryGetResults(resultsList, pageSize);
+
+        when(catalogFramework.query(any())).thenReturn(queryResponseMock);
+        when(queryResponseMock.getResults()).thenReturn(queriedResultsList[0], queriedResultsList);
```

Comment:
@josephthweatt This is something that Richard _did_ like along the lines of internal query calls being abstracted out to be more readable: https://github.com/codice/ddf/blob/b132cafeb6b770730c0c243dcd988f57b0580b36/catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveAllCommandTest.java#L123

😄